### DISCUSSION TITLE
Information display

### DISCUSSION THREAD

**Discussion by Queen-esther01** [heroui-inc/heroui#282] [Feedback]
Really going through NextUI and It's definitely something I would like to try out. I did notice two things missing: Alerts and Toasts. I use these two components a lot in any library I use and I wonder if this is part of the coming upgrades. 

Anyways, Kudos to the team behind this!

**Comment by jrgarciadev** [comment]
Hey @Queen-esther01, sure, those components are in the backlog 👍🏻

**Reply by Queen-esther01** [reply]
Great!

**Reply by Queen-esther01** [reply]
I created two issues for this, I hope this is not a problem. I can go delete them if it is.
